.github/workflows/publish-image.yml

```diff
@@ -0,0 +1,35 @@
+name: publish-to-quay
+on:
+  push:
+    branches:
+      - master
+env:
+  REPOSITORY: quay.io/refgenomics/covid19
+  TAG: latest
+
+jobs:
+  publish-to-quay:
+    runs-on: ubuntu-latest
+    environment: production
```

Comment:
This ensures that the action is run within the `production` environment which contains the secrets. The environment has additional security rules in place (the CI action needs approval from select users).